iggy/Cargo.toml

```diff
@@ -5,7 +5,7 @@ description = "Iggy is the persistent message streaming platform written in Rust
 edition = "2021"
 license = "MIT"
 keywords = ["iggy", "messaging", "streaming"]
-categories = ["messaging", "streaming"]
+categories = ["database", "network-programming"]
```

Comment:
Since this is the only crate from whole project workspace which is published to crates.io perhaps we could consider adding also command-line-utilities (iggy server and iggy cmd crates)?

Reply:
Sure, done :)